schema/proto3/subscribeinfoprovisioner.proto

```diff
@@ -0,0 +1,43 @@
+/*
+MIT License
+Copyright(c) 2020 Futurewei Cloud
+
+    Permission is hereby granted,
+    free of charge, to any person obtaining a copy of this software and associated documentation files(the "Software"), to deal in the Software without restriction,
+    including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and / or sell copies of the Software, and to permit persons
+    to whom the Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+    WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+syntax = "proto3";
+
+package alcor.schema;
+
+option java_package = "com.futurewei.alcor.schema";
+
+service SubscribeInfoProvisioner {
+
+  rpc PushNodeSubscribeInfo (NodeSubscribeInfo) returns (SubscribeOperationReply) {
+  }
+
+}
+
+enum SubscribeOperation {
+  Subscribe = 0;
+  Unsubscribe = 1;
+}
+
+message NodeSubscribeInfo {
+  SubscribeOperation subscribe_operation = 1;
+  string topic = 2;
+  string key = 3;
```

Comment:
What does the key differ from topic?

Reply:
The key is for key-shared mode in Pulsar. It's similar to the concept of a secondary topic in the message queue (e.g. key in Apache Pulsar, tag in RocketMQ). At the same time, this value can be set to null when it is not needed.

Reply:
It would be great to put the comment of "Allow to set NULL when a secondary topic is not needed" for the key field. @VanderChen